Random failing doctests

I have run the testsuite multiple times and some times it fails (and sometime it doesnt) due to:

```
======================================================================
FAIL: Doctest: scrapely.extraction.regionextract.RecordExtractor
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/doctest.py", line 2226, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for scrapely.extraction.regionextract.RecordExtractor
  File "/home/daniel/src/scrapely/scrapely/extraction/regionextract.py", line 291, in RecordExtractor

----------------------------------------------------------------------
File "/home/daniel/src/scrapely/scrapely/extraction/regionextract.py", line 306, in scrapely.extraction.regionextract.RecordExtractor
Failed example:
    ex.extract(page)
Expected:
    [{u'description': [u'description'], u'name': [u'name']}]
Got:
    [{u'name': [u'name'], u'description': [u'description']}]


----------------------------------------------------------------------
Ran 99 tests in 0.578s

FAILED (failures=1)
ERROR: InvocationError: '/home/daniel/src/scrapely/.tox/py27/bin/nosetests scrapely tests'
_______________________________________________________________________________________________ summary _______________________________________________________________________________________________
ERROR:   py27: commands failed
```

--- a/scrapely/extraction/regionextract.py
+++ b/scrapely/extraction/regionextract.py
@@ -306,8 +306,8 @@ class RecordExtractor(object):
             u'<h1>name</h1> <p>description</p>')
     >>> basic_extractors = map(BasicTypeExtractor, template.annotations)
     >>> ex = RecordExtractor.apply(template, basic_extractors)[0]
-    >>> ex.extract(page)
-    [{u'description': [u'description'], u'name': [u'name']}]
+    >>> ex.extract(page) == [{u'description': [u'description'], u'name': [u'name']}]
+    True
     """
     
     def __init__(self, extractors, template_tokens):

